server/schedulers/grant_leader.go

```diff
@@ -308,7 +308,7 @@ func (handler *grantLeaderHandler) DeleteConfig(w http.ResponseWriter, r *http.R
 		return
 	}
 
-	handler.rd.JSON(w, http.StatusNotFound, ErrScheduleConfigNotExist)
+	handler.rd.JSON(w, http.StatusNotFound, errs.ErrScheduleConfigNotExist.FastGenByArgs())
```

Comment:
Should call `.Error()` explicitly here or you will get an empty object returned instead of a string

Reply:
Good, I will fix it in new PR